src/components/SRNEntry.vue

```diff
@@ -105,56 +105,71 @@ export default {
     return {
       userIDList: [{ userID: "", valid: false }], //array containing user-id's and a valid flag for each
       invalidInputMessage: null, //flag to check if the input is in correct format
-      isUserValid: false, // whether the user exists in the backend database
+      isCurrentUserValid: false, // temp flag which contains the validity of the most recent user ID
       maxLengthOfSRN: 10,
       validateCount: 0, //this variable tells us how many times the user has been validated.
       invalidLoginMessage: "Please enter correct SRN / कृपया सही SRN दर्ज करें",
-      isLoading: false,
+      isLoading: false, 
     };
   },
   computed: {
+    //returns length of the userID list
     userIdListLength() {
       return this.userIDList.length;
     },
+    //checks if any userID has been typed
     isAnyUserIDPresent() {
-      //checks if any userID has been typed
       return (
         this.userIDList != undefined &&
         this.userIdListLength > 0 &&
         this.userIDList[0]["userID"] != ""
       );
     },
+    //checks if multiple entries are allowed and if one entry is already inputted
     ifUserEnteredMoreThanOne() {
-      //used when multiple SRN's will be allowed to typed
       return !this.isSingleEntryOnly && this.userIdListLength > 1;
     },
+    //for now, plio does not support multiple input entries
     isSingleEntryOnly() {
       return this.redirectTo == "plio";
     },
+    /* returns submit button as disabled if any of the following is true:
+     - no SRN has been typed
+     - input is invalid
+     - SRN hasn't been completely typed */
     isSubmitButtonDisabled() {
-      //submit button is disabled if no SRN has been typed or input is invalid or SRN hasn't been completely typed
       return (
         !this.isAnyUserIDPresent ||
         this.invalidInputMessage != "" ||
         this.isCurrentEntryIncomplete
       );
     },
+    /* checks if + button should be displayed. Will be activated only if:
+     - multiple entries are allowed
+     - if current input entry is complete
+     - if cap of maximum entries hasn't been reached yet */
     isAddButtonAllowed() {
-      //if multiple SRN's are allowed, then this helps with the activation of the + button, to add more SRN's
       return (
         !this.isSingleEntryOnly &&
         !this.isCurrentEntryIncomplete &&
         this.userIdListLength < numberOfSRNsAllowed
       );
     },
+    // checks if the current input entry has the required number of characters.
     isCurrentEntryIncomplete() {
       return this.getLatestEntry["userID"].length < this.maxLengthOfSRN;
     },
+    // returns the most recent typed input entry
     getLatestEntry() {
```

Comment:
`get...` is a name for a method. Since this is a computed property, the more appropriate name is `latestEntry`:

```suggestion
    // returns the most recently entered input
    latestEntry() {
```